README.md

```diff
@@ -127,6 +127,18 @@ jakartaeeMigration {
 }
 ```
 
+### Including Artifacts to Transformation
+
+Artifacts can be included to transforms using `includeTransform`:
+```
+jakartaeeMigration {
+    // These artifacts have references to 'javax' and must be transformed
```

Comment:
Actually it's the opposite for those specific artifacts - the `javax` references are intentional and should be left alone.